Documentation/operations/upgrade.rst

```diff
@@ -901,7 +921,7 @@ New ConfigMap Options
 
     The option is enabled by default for new deployments when generated via
     Helm, in order to gain the benefits of improved security. The Helm option
-    is ``--set global.remoteNodeIdentity``. This option can be disabled in
+    is ``--set remoteNodeIdentity``. This option can be disabled in
```

Comment:
this should not be modified since this section referes to upgrade from 1.7 to 1.8, we aren't changing the helm chart of 1.8 branch.

Reply:
fixed